travis-ci/vmtest/configs/blacklist/BLACKLIST-latest

```diff
@@ -1,7 +1,6 @@
 # TEMPORARILY DISABLED
 send_signal		# flaky
 test_lsm		# semi-working
-sk_assign		# needs better setup in Travis CI
 sk_lookup
```

Comment:
can you try enabling sk_lookup as well? I'm not sure, but it might have been disabled due to the same iproute2 reason.